tests/lib/fakeinstaller/go.mod

```diff
@@ -0,0 +1,6 @@
+module github.com/snapcore/snapd/tests/lib/muinstaller
+
+go 1.18
+
+require github.com/snapcore/snapd v0.0.0-20220929103851-d41483655caf
```

Comment:
Maybe it is possible to use the local code by adding:
```
replace github.com/snapcore/snapd => /snapd-local-go-path
```

Reply:
But looking in more detail this go.mod is used only when building the snap and it is not really used for the tests, so I think it is fine to have a concrete commit here.